examples/undocumented/python/kernel_combined_custom_poly.py

```diff
@@ -9,22 +9,22 @@
 
 def kernel_combined_custom_poly (train_fname = traindat,test_fname = testdat,train_label_fname=label_traindat):
     from shogun import CombinedFeatures, RealFeatures, BinaryLabels
-    from shogun import CombinedKernel, PolyKernel, CustomKernel
+    from shogun import CustomKernel
     from shogun import CSVFile
     import shogun as sg
 
-    kernel = CombinedKernel()
+    kernel = sg.CombinedKernel()
     feats_train = CombinedFeatures()
 
     tfeats = RealFeatures(CSVFile(train_fname))
-    tkernel = PolyKernel(10,3)
+    tkernel = sg.kernel("PolyKernel", cache_size=10, degree=3)
```

Comment:
No need to have the cache size... but don’t bother changing... just for next pr

Reply:
OK!